FantasmoSDK/app/src/main/java/com/fantasmo/sdk/network/FMNetworkManager.kt

```diff
@@ -237,27 +233,34 @@ class FMNetworkManager(
         } else {
             val errorResult = String(networkResponse.data)
             try {
-                val response = Gson().fromJson(errorResult, ErrorResponse::class.java)
+                response = Gson().fromJson(errorResult, ErrorResponse::class.java)
+                val debugMessage = response.message ?: response.detail ?: ""
 
                 when (networkResponse.statusCode) {
                     404 -> {
                         errorMessage = "Resource not found"
                     }
+                    403 -> {
+                        errorMessage = "Forbidden"
+                    }
                     401 -> {
-                        errorMessage = "${response.message} Authentication error"
+                        errorMessage = "$debugMessage Authentication error"
```

Comment:
don't these need `${curlyBrackets}` to interpolate properly?

Reply:
The brackets are only needed if the thing one wants to print is more than just a variable name